respy/simulate.py

```diff
@@ -14,23 +16,33 @@
 from respy.state_space import StateSpace
 
 
-def simulate(params, options):
-    """Simulate a data set.
+def get_simulate_func(params, options):
+    """Get the criterion function.
 
-    This function provides the interface for the simulation of a data set.
+    Return v version of the likelihood functions in respy where all arguments
```

Comment:
Here you copy pasted a bit too much from the `get_crit_func` docstring.

I think it would also be good if the docstring explains why we need it and can't call the function directly. I.e. the speed benefit for a MSM Simulation.

Reply:
Good catch! Forgot that.